ics27_config_example.toml

```diff
@@ -0,0 +1,51 @@
+[global]
```

Comment:
I would be in favor of not introducing another configuration file. I can see why it would be useful to have this ICS27-specific one around, however. We could improve things if we tidy up our config files and put all of them in a separate top-level folder.